AblyAssetTracking.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -172,6 +172,15 @@
           "version": "3.1.1"
         }
       },
+      {
+        "package": "swift-argument-parser",
```

Comment:
I don't see any uses of `import ArgumentParser`, I'm wondering if this dependency is needed or accidental?

Reply:
it was added before my joining this project, and to be honest, I don't know what is it for. I'll investigate it and will remove it if it's unnecessary. But it should be done in separate PR

An Issue: https://github.com/ably/ably-asset-tracking-swift/issues/211